api/v1beta1/agent_types.go

```diff
@@ -236,6 +236,10 @@ type AgentStatus struct {
 	// ValidationsInfo is a JSON-formatted string containing the validation results for each validation id grouped by category (network, hosts-data, etc.)
 	// +optional
 	ValidationsInfo common.ValidationsStatus `json:"validationsInfo,omitempty"`
+
+	// InstallationDiskID defines the installation destination disk (must be equal to the inventory disk id).
```

Comment:
This comment doesn't make as much sense for the status.
```suggestion
	// InstallationDiskID is the disk that will be used for the installation.
```